### ISSUE TITLE
App freezes after clicking "send file" button

### ISSUE THREAD

**Issue by ZerdoX-x** [telegramdesktop/tdesktop#16377]
### Steps to reproduce
1. Launch telegram-desktop.
2. Select any chat.
3. Click "send flie" button or press `Ctrl+O` shortcut.

### Expected behaviour
I can see my files immediately after clicking the button.

### Actual behaviour
App freezes (even caret is not blinking).
I need to wait about 20 seconds till window with my files opens.

### Configuration
**Operating system:** 5.12.6-arch1-1

**Version of Telegram Desktop:** 2.7.4

**Installation source:** pacman (https://archlinux.org/packages/community/x86_64/telegram-desktop/)

**Used theme:** N/A

**Logs:** I've tried launching telegram-desktop manually from terminal and no logs appeared after I reproduced the bug.

### Additional info
<details><summary>Window with my files looks different from same window using firefox for example. Here is some screenshots:</summary>

Window opened by telegram:
![image](https://user-images.githubusercontent.com/49815452/119804855-2d893400-befa-11eb-9478-3e9fb4cd1f8c.png)
Window opened by firefox:
![image](https://user-images.githubusercontent.com/49815452/119805113-632e1d00-befa-11eb-8961-188d5ead0297.png)
Window opened by vscode (same as firefox):
![image](https://user-images.githubusercontent.com/49815452/119805361-9ec8e700-befa-11eb-9de8-0d19794e7ccb.png)

</details>

So all apps that I use and tested (firefox, chromium, vscode, discord, figma-linux) use the same window.
Telegram's one is different. Maybe some external API is broken?

**Comment by ZerdoX-x** [comment]
I can paste full logs in some external resources in debug mode if needed.

**Comment by ZerdoX-x** [comment]
```
TDESKTOP_USE_GTK_FILE_DIALOG=1
```
Makes window appearance same as in other apps but freezing and delay is the same.

**Comment by Aokromes** [comment]
linux distribution issue, try downloading from desktop.telegram.org

**Comment by ZerdoX-x** [comment]
@Aokromes Please reopen. Issue is not resolved yet :-||

I have installed binary from (desktop.telegram.org)[https://dekstop.telegram.org]. Nothing changed.
I guess this is not related to installation source.

**Comment by Aokromes** [comment]
> @Aokromes Please reopen. Issue is not resolved yet :-||
> 
> I have installed binary from (desktop.telegram.org)[https://dekstop.telegram.org]. Nothing changed.
> I guess this is not related to installation source.
> 
> I also would like to notice again that any other software works and interacts with system fine.

you have download 2.7.4 or 2.7.5?

**Comment by ZerdoX-x** [comment]
> you have download 2.7.4 or 2.7.5?

2.7.4

**Comment by Aokromes** [comment]
> > you have download 2.7.4 or 2.7.5?
> 
> 2.7.4
> 
> I also noticed same problem when I am trying to open "Settings > Chat settings". App freezes and only after about 20 seconds I can see settings. I don't know if this related. But there is difference between problems: Reopening chat settings do reproduce bug but reopening files window does not. If nothing will be moved on I would like to open third issue about chat settings. Won't do it now because these two problems and #16376 may be cause by one single bug.
> 
> UPD: All other settings, pages and ui blocks work fine except:
> 
> * Opening app
> * Chat settings
> * Attaching file

try 2.7.5 beta to see if it changes something.

**Comment by ZerdoX-x** [comment]
2.7.5 File attaching works fine! Then I will open another issue about chat settings.

Thank you
